src/diff/amfGraphDifferencer.test.ts

```diff
@@ -68,7 +68,7 @@ describe("Validate flattened AMF graph", () => {
 describe("Changes to graph nodes", () => {
```

Comment:
Chai recommends using `lengthOf` instead of `length`: https://www.chaijs.com/api/bdd/#method_lengthof